test-configs.yaml

```diff
@@ -1876,6 +1891,7 @@ test_configs:
       - baseline
       - baseline-nfs
       - igt-kms-exynos
+      - ltp-crypto
```

Comment:
I thought this was found to be failing on odroid-xu3 as per the table on #506?

Reply:
It failed initially, I re-ran failed cases found its passed. Updated table can be found here https://github.com/kernelci/kernelci-core/issues/506#issuecomment-735860447

Reply:
It's failing again now because ODROID-XU3 is failing to boot with mainline, next and even stable 5.4.  So we should drop it from this PR and potentially add it later when support for this platform has been fixed in the kernel.